espnet2/tasks/svs.py

```diff
@@ -265,7 +272,8 @@ def build_preprocess_fn(
                 g2p_type=args.g2p,
                 fs=args.fs,
                 time_shift=args.feats_extract_conf["hop_length"]
-                / args.feats_extract_conf["fs"],
+                # / args.feats_extract_conf["fs"], # bug
```

Comment:
Any clue for this one? We should have the argument in feats_extract

Reply:
Given that, we would have fs anyway, I do not think we need to call the divide here. @jerryuhoo could you reflect the calculation at https://github.com/espnet/espnet/blob/master/espnet2/train/preprocessor.py#L994-L995 and remove the line here ?